test/dlc_tests/conftest.py

```diff
@@ -908,20 +908,55 @@ def skip_transformer_engine_test(request):
 
 
 @pytest.fixture(autouse=True)
-def skip_pt20_cuda121_tests(request):
+def skip_smdebug_test(request):
     if "training" in request.fixturenames:
         img_uri = request.getfixturevalue("training")
     elif "pytorch_training" in request.fixturenames:
         img_uri = request.getfixturevalue("pytorch_training")
     else:
         return
-    if request.node.get_closest_marker("skip_pt20_cuda121_tests"):
-        _, image_framework_version = get_framework_and_version_from_tag(img_uri)
-        image_processor = get_processor_from_image_uri(img_uri)
-        image_cuda_version = get_cuda_version_from_tag(img_uri)
-        if Version(image_framework_version) in SpecifierSet("==2.0.1") and image_processor == "gpu":
-            if Version(image_cuda_version.strip("cu")) == Version("121"):
-                pytest.skip("PyTorch 2.0 + CUDA12.1 image doesn't support current test")
+    _, image_framework_version = get_framework_and_version_from_tag(img_uri)
+    image_processor = get_processor_from_image_uri(img_uri)
+    image_cuda_version = get_cuda_version_from_tag(img_uri)
+    if request.node.get_closest_marker("skip_smdebug_test"):
+        if (
+            Version(image_framework_version) in SpecifierSet("==2.0.1")
```

Comment:
This looks like a pretty strict condition. Is it safe to specify btwn 2.0 and 2.1, in case we release patch versions like 2.0.2 in the future?

Can update such that `SpecifierSet("==2.0.*")`